### PR TITLE
Rocky9 RPM Support

### PR DESCRIPTION
**Pull request by TheBurchLog**

### CODE REVIEW

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -24,4 +24,26 @@ jobs:
...
-        run: make build-image-python${{ matrix.python-version }}
+        run: make build-image-centos7-python${{ matrix.python-version }}
+
+  docker-publish-rocky9-python:
+    name: Docker Publish bgio/build:rocky9-python${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.7.13", "3.8.18", "3.9.18","3.10.13","3.11.6","3.12.7"]
```
Comment (justinryan1) [inline .github/workflows/docker_build.yml]: Can we change 3.11.6 to 3.11.10?

File: docker/dockerfiles/build/Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 SHELL:=/bin/bash
 
-build-all: | build-image-base build-image-python3.8.18 build-image-python3.9.18 build-image-python3.10.13 build-image-python3.11.6 build-image-python3.12.0
+build-all-centos7: | build-image-base build-image-centos7-python3.8.18 build-image-centos7-python3.9.18 build-image-centos7-python3.10.13 build-image-centos7-python3.11.6 build-image-centos7-python3.12.0
```
Comment (justinryan1) [inline docker/dockerfiles/build/Makefile]: Can we change 3.11.6 to 3.11.10 and 3.12.0 to 3.12.7?

File: docker/dockerfiles/build/Makefile [hunk 3]
```diff
@@ -62,4 +62,65 @@ build-image-python3.13.0:
 	docker push bgio/build:centos7-python3.13; \
 	docker push bgio/build:centos7-python3.13.0;
 
+build-all-rocky9: | build-image-rocky9-python3.7.13 build-image-rocky9-python3.8.18 build-image-rocky9-python3.9.18 build-image-rocky9-python3.10.13 build-image-rocky9-python3.11.6 build-image-rocky9-python3.12.7
```
Comment (justinryan1) [inline docker/dockerfiles/build/Makefile]: Can we change 3.11.6 to 3.11.10?

File: .github/workflows/docker_build.yml [hunk 4]
```diff
@@ -24,4 +24,26 @@ jobs:
...
-        run: make build-image-python${{ matrix.python-version }}
+        run: make build-image-centos7 PYTHON_VERSION=${{ matrix.python-version }}
+
+  docker-publish-rocky9-python:
+    name: Docker Publish bgio/build:rocky9-python${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.8.18", "3.9.18","3.10.13","3.11.10","3.12.7","3.13.0"]
```
Comment (1maple1) [inline .github/workflows/docker_build.yml]: random space

File: docker/dockerfiles/build/rocky9/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        make \
+        bzip2-devel \
+        sqlite-devel \
+        perl-core \
+        pcre-devel \
+        wget \
+        zlib-devel \
+        openssl-devel
+
+
```
Comment (1maple1) [inline docker/dockerfiles/build/rocky9/Dockerfile]: extra space

**Pull request merged**